### PR TITLE
Fix Changelog after merging an unreleased release branch.

### PR DESCRIPTION
**Pull request by micbar**
## Description
- The 10.4 release branch was merged to master before we finished the release
- We need to remove the 10.4.0 Version from master

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Review by jnweiger — APPROVED** [review]
this fixes the changelog, without touching any code.
36727 36726 36722 36717 36709 were accidentially copied from subfolders unreleased to 10.4.0_2020-01-10
All older issue 10.4 issue numbers were accidentially moved from subfolder unreleased to 10.4.0_2020-01-10

This PR reverts both. Sorry for the noise. Release is close, but not done yet.

**Comment by jnweiger** [comment]
https://drone.owncloud.com/owncloud/core/22705/54/5 failed with an infrastructure error
`apiTrashbin-mariadb10.2-php7.1: Error: No such image: docker.io/owncloudci/php:7.1`

Restarting drone/pr as https://drone.owncloud.com/owncloud/core/22706

File: changelog/10.4.0_2020-01-10/36709 [hunk 1]
```diff
@@ -1,3 +0,0 @@
-Change: Update league/flysystem (1.0.62 => 1.0.63)
```
Comment (phil-davis) [inline changelog/10.4.0_2020-01-10/36709]: Note: these are OK to delete. They also exist in the changelog/unreleased folder already

**Comment by phil-davis** [comment]
> codecov tests are pending for over 1h now. Status unclear to me.

@jnweiger codecov is a strange beast - the codecov report finally arrived a few minutes ago!

**Comment by micbar** [comment]
:+1 for Codacy

**Pull request merged**